### PR TITLE
Move Alacritty to organization

### PR DESCRIPTION
**Pull request by chrisduerr**
This fixes various outdated links pointing to the old jwilm/alacritty
repository.

Since `copypasta` now has its own github repository at
https://github.com/alacritty/copypasta, the sources have been removed
from Alacritty.

### CODE REVIEW

File: alacritty/Cargo.toml [hunk 1]
```diff
@@ -1,11 +1,11 @@
 [package]
 name = "alacritty"
 version = "0.4.2-dev"
-authors = ["Joe Wilm <joe@jwilm.com>"]
+authors = ["Christian Duerr <contact@christianduerr.com", "Joe Wilm <joe@jwilm.com>"]
```
Comment (nixpulvis) [inline alacritty/Cargo.toml]: Bout time. ;)

File: alacritty_terminal/Cargo.toml [hunk 2]
```diff
@@ -24,7 +24,7 @@ unicode-width = "0.1"
 base64 = "0.11.0"
 terminfo = "0.6.1"
 url = "2"
-copypasta = { path = "../copypasta" }
+copypasta = "0.6.1"
```
Comment (nixpulvis) [inline alacritty_terminal/Cargo.toml]: :tada:

**Comment by kchibisov** [comment]
Please can we keep our font crate in alacritty for a while?

**Comment by chrisduerr** [comment]
> Please can we keep our font crate in alacritty for a while?

Yeah, I think that makes sense. I see no rush to move it since I'm not aware of any interest for that outside of the Alacritty repository.

And having to manually apply your patches to it sounds like a massive mess. :D

File: alacritty/Cargo.toml (hunk 1 above)
Comment (kchibisov) [reply alacritty/Cargo.toml]: We can put this line in front `The alacritty contributors`, like winit is doing btw.

**Comment by nixpulvis** [comment]
> I mean it'll break some big existing PRs, etc(Windows fallback/fontconfig fallback)

Good point, and this PR should ideally land ASAP since the repo has in fact already moved.

**Pull request merged**